### PR TITLE
Redirect finetuning_torchvision_models_tutorial to torchvision_tutorial

### LINKED ISSUE #2097: broken link to normalization values
## 📚 Documentation

There is a broken [link](https://pytorch.org/docs/master/torchvision/models.html) in the Load Data section of [this](https://pytorch.org/tutorials/beginner/finetuning_torchvision_models_tutorial.html) page.

cc @svekars @carljparker @datumbox @nairbv @fmassa @NicolasHug @YosuaMichael

### PR DESCRIPTION
**Pull request by tcNickolas**
Fixes #2097

## Checklist

- [x] The issue that is being fixed is referred in the description (see above "Fixes #ISSUE_NUMBER")
- [x] Only one issue is addressed in this pull request
- [x] Labels from the issue that this PR is fixing are added to this pull request
- [x] No unnecessary issues are included into this pull request.

cc @svekars @carljparker @datumbox @nairbv @fmassa @NicolasHug @YosuaMichael

### CODE REVIEW

**Review by svekars — APPROVED** [review]
Thanks!
